apps/ima/behaviours/reset-human-rights-summary.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
```

Comment:
The `locals` function is for loading any data required for rendering the page template so I don't think it should be used here - try using a `configure` function if importing the behaviour in the` human-rights-family-summary` step or a `saveValues` function if importing the behaviour in the `human-rights-claim` step. 

I wouldn't import the behaviour in the `other-human-rights-claims` step as you are giving the responsibility of retrospectively unsetting information on human-rights-family-summary to a step that to which it has no dependency. As human-rights-family-summary is dependent on `human-rights-claim` being answered yes and the subsequent questions being filled out I would import the behaviour on one of those two steps.

Reply:
I've changed the behaviour to now use the `saveValues` function and imported it in the `human-rights-claim` page instead of the `other-human-rights-claims` page.